tests/test_lsst_efd_client.py

```diff
@@ -205,6 +205,11 @@ async def test_top_n(efd_client, start_stop):
     assert len(df) == 10
     for c in ['foo', 'bar']:
         assert c in df.columns
+    df = await efd_client.select_top_n('lsst.sal.fooSubSys.test', ['foo', 'bar'], 10, time_cut=start_stop[0])
+    assert len(df) == 10
+    for c in ['foo', 'bar']:
+        assert c in df.columns
+    assert df['foo'].values[0] == 144.11835565266966
```

Comment:
If you're not going to check them all, maybe check first and last?  I don't really have a strong opinion.